extensions/positron-python/src/test/positron/manager.unit.test.ts

```diff
@@ -196,3 +199,222 @@ suite('Python runtime manager', () => {
         verify(mockedPositronNamespaces.runtime!.selectLanguageRuntime(runtimeMetadata.object.runtimeId)).once();
     });
 });
+
+suite('Python runtime manager - recommendedWorkspaceRuntime', () => {
```

Comment:
It might be worth checking with QA whether they think these would make good e2e tests since interpreter discovery/startup is so important and pretty complex.

Reply:
Agreed 💯 After working on this PR, I noticed a lot of weirdness around interpreter selection; the suggested interpreter was not the one selected, a global pyenv being selected over other options (which should not happen with our sorting), etc. 


I opened up a new issue #6678 to take a closer look at interpreter selection and sort. More testing, likely e2e, would be part of the acceptance criteria for closing that out.